eng/pipelines/eng-common-sync.yml

```diff
@@ -36,7 +36,7 @@ pr:
       - eng/common
 
 pool:
-  vmImage: windows-2019
+  vmImage: ubuntu-18.04
```

Comment:
Is there a particular reason for this switch?

Reply:
Java long paths, I will switch it back once that is fixed